### PR TITLE
Edit Feature

### PR DESCRIPTION
**Pull request by anandsr21harshit**
- user can now edit notes
- made whole UI fully responsive

### CODE REVIEW

File: keep-notes/src/components/NavBar.jsx [hunk 1]
```diff
@@ -1,12 +1,65 @@
...
   return (
-    <nav className="nav-bar">
-      <h1 className="nav-brand">Keep Notes</h1>
-    </nav>
+    <>
+      <nav className="nav-bar">
+        <h1 className="nav-brand">Keep Notes</h1>
+        <ul class="menus">
+          <li class="menu">
+            <i class="bi bi-list" onClick={() => setList(!list)}></i>
```
Comment (anuj-kumary) [inline keep-notes/src/components/NavBar.jsx]: Change class to className

**Review by anuj-kumary — APPROVED** [review]
- Code looks clean
- Good folder structure

File: keep-notes/src/components/NoteCard.jsx [hunk 2]
```diff
@@ -15,57 +17,86 @@ function NoteCard({ notesData }) {
...
-      if(response.status === 200 || response.status === 201){
-        const deletedData = state.notes.filter(item => item._id === notesData._id)[0];
-        console.log({deletedData});
-        dispatch({type:"TRASH", payload:deletedData})
-        dispatch({type:"ADD_NOTES",payload:{note: response.data.notes}})
+      if (response.status === 200 || response.status === 201) {
+        const deletedData = state.notes.filter(
+          (item) => item._id === notesData._id
+        )[0];
+        console.log({ deletedData });
```
Comment (gopalbbharadva) [inline keep-notes/src/components/NoteCard.jsx:24]: Remove consoles.

File: keep-notes/src/components/NotesInput.jsx [hunk 3]
```diff
@@ -22,19 +21,32 @@ function NotesInput() {
...
+          }
+        );
+      } else {
+        response = await axios.post(
+          "/api/notes",
+          { note },
+          { headers: { authorization: token } }
+        );
+      }
       console.log(response);
```
Comment (gopalbbharadva) [inline keep-notes/src/components/NotesInput.jsx:50]: Remove console.

**Review by gopalbbharadva — APPROVED** [review]
Code is well-structured, One suggestion: provide live link so your feature in UI will also  be reviewed.

**Review by mdarif6 — APPROVED** [review]
- Liked folder structure- clean 
- convert class to className
- need to see UI.

File: keep-notes/src/components/NoteCard.jsx (hunk 2 above)
Comment (anandsr21harshit) [reply keep-notes/src/components/NoteCard.jsx:24]: yes will refactor that

File: keep-notes/src/components/NavBar.jsx (hunk 1 above)
Comment (anandsr21harshit) [reply keep-notes/src/components/NavBar.jsx]: will change it

**Review by manojasarsa — APPROVED** [review]
The code seems clean.
Good to merge

**Pull request merged**